src/gdprApi/__tests__/useAuthCodeQueues.test.tsx

```diff
@@ -535,13 +489,15 @@ describe('useAuthCodeQueues', () => {
     it('when storage has a queue with failed auth code retrieval, nextPhase is "restart"', async () => {
       mockedWindowControls.setPath(config.downloadPath);
       mockedWindowControls.setSearch(
-        createFailedActionParams(tunnistamoAuthCodeCallbackUrlAction as Action)
+        // createFailedActionParams(tunnistamoAuthCodeCallbackUrlAction as Action)
```

Comment:
I guess this commented line should be removed.

Reply:
removed